reactive/filebeat.py

```diff
@@ -99,6 +99,9 @@ def render_filebeat_template():
         # NB: beat base layer will handle setting a waiting status
         service("stop", "filebeat")
 
+    # run the sanity checks
+    sanity_check()
+
```

Comment:
please sanity check the config as early as possible. If you do it this late it cannot prevent the on-disk configuration from being rendered, meaning that an accidental restart of filebeat would cause it to load values that would fail the check